backend/django/core/views/smart_embeddings_model/README.md

```diff
@@ -0,0 +1,87 @@
+---
+pipeline_tag: sentence-similarity
+tags:
+- sentence-transformers
+- feature-extraction
+- sentence-similarity
+---
+
+# {MODEL_NAME}
+
+This is a [sentence-transformers](https://www.SBERT.net) model: It maps sentences & paragraphs to a 384 dimensional dense vector space and can be used for tasks like clustering or semantic search.
+
+<!--- Describe your model here -->
```

Comment:
Seeing these comments here. It looks like this is a template. Do we want to fill these in?

Reply:
Yea we can fill the README out if we want!